configs/droid.kmap

```diff
@@ -22,3 +22,4 @@ keycode 226 = ToggleCallHangup
 keycode 256 = ToggleCallHangup
 keycode 528 = CameraFocus
 keycode 766 = Camera
+keycode 457 = Camera
```

Comment:
Could you please re-add the newline at the end of the file? (One sure way to do this is to open this file with `vim`, add a character, delete that character, then save the file:)

Reply:
Done.